Makefile

```diff
@@ -98,7 +98,7 @@ install_nodejs_sdk:
 	yarn link --cwd $(WORKING_DIR)/sdk/nodejs/bin
 
 install_plugins: .pulumi/bin/pulumi
-	.pulumi/bin/pulumi plugin install resource aws 5.37.0
+	.pulumi/bin/pulumi plugin install resource aws 6.8.0
```

Comment:
# WARNING: This file is autogenerated - changes will be overwritten if not made via https://github.com/pulumi/ci-mgmt